.github/workflows/test.yml

```diff
@@ -7,22 +7,27 @@ jobs:
     strategy:
       matrix:
         ocaml-version: [ '4.08.1', '4.09.0', '4.10.0' ]
-        operating-system: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
why remove Windows from this? All the packages should compile on Cygwin too (since mirage-crypto got ported)